cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -111,7 +111,7 @@ You'll need this software installed:
 Java Runtime
 ++++++++++++
 The latest `JDK or JRE version 1.7.xx <http://www.java.com/en/download/manual.jsp>`__
-for Linux and Solaris must be installed in your environment; we recommend the Oracle JDK.
+for Linux, Windows, or Mac OS X must be installed in your environment; we recommend the Oracle JDK.
```

Comment:
Why did you drop Solaris? While we don't have packages for Solaris, CDAP can run on Solaris, just fine. All it needs are tools like `bash` and `curl` which are available on Solaris, and `node` and JDK.